letsencrypt/build.yaml

```diff
@@ -1,37 +1,38 @@
 ---
 build_from:
-  aarch64: ghcr.io/home-assistant/aarch64-base-python:3.11-alpine3.18
-  amd64: ghcr.io/home-assistant/amd64-base-python:3.11-alpine3.18
-  armhf: ghcr.io/home-assistant/armhf-base-python:3.11-alpine3.18
-  armv7: ghcr.io/home-assistant/armv7-base-python:3.11-alpine3.18
-  i386: ghcr.io/home-assistant/i386-base-python:3.11-alpine3.18
+  aarch64: "ghcr.io/home-assistant/aarch64-base-python:3.11-alpine3.18"
+  amd64: "ghcr.io/home-assistant/amd64-base-python:3.11-alpine3.18"
+  armhf: "ghcr.io/home-assistant/armhf-base-python:3.11-alpine3.18"
+  armv7: "ghcr.io/home-assistant/armv7-base-python:3.11-alpine3.18"
+  i386: "ghcr.io/home-assistant/i386-base-python:3.11-alpine3.18"
 codenotary:
-  signer: notary@home-assistant.io
-  base_image: notary@home-assistant.io
+  signer: "notary@home-assistant.io"
+  base_image: "notary@home-assistant.io"
 args:
-  CLOUDFLARE_VERSION: 2.9.11
-  CRYPTOGRAPHY_VERSION: 41.0.5
-  CERTBOT_VERSION: 2.7.4
-  CERTBOT_DNS_AZURE_VERSION: 2.4.0
-  CERTBOT_DNS_CLOUDNS_VERSION: 0.6.0
-  CERTBOT_DNS_DESEC_VERSION: 1.2.1
-  CERTBOT_DNS_DIRECTADMIN_VERSION: 1.0.3
-  CERTBOT_DNS_DUCKDNS_VERSION: 1.3
-  CERTBOT_DNS_DYNU_VERSION: 0.0.5
-  CERTBOT_DNS_EASYDNS_VERSION: 0.1.2
-  CERTBOT_DNS_HE_VERSION: 1.0.0
-  CERTBOT_DNS_HETZNER_VERSION: 2.0.0
-  CERTBOT_DNS_INFOMANIAK_VERSION: 0.2.1
-  CERTBOT_DNS_INWX_VERSION: 2.2.0
-  CERTBOT_DNS_JOKER_VERSION: 1.1.0
-  CERTBOT_DNS_NAMECHEAP_VERSION: 1.0.0
-  CERTBOT_DNS_GOOGLE_DOMAINS_VERSION: 0.1.11
-  CERTBOT_DNS_TRANSIP_VERSION: 0.5.2
-  CERTBOT_DNS_PORKBUN_VERSION: 0.8.0
-  CERTBOT_DNS_WEBSUPPORT_VERSION: 2.0.1
-  CERTBOT_GANDI_VERSION: 1.5.0
-  CERTBOT_NETCUP_VERSION: 1.4.3
-  CERTBOT_NJALLA_VERSION: 1.0.0
-  CERTBOT_DNS_DREAMHOST_VERSION: 1.0
-  CERTBOT_DNS_DOMAINOFFENSIVE_VERSION: 2.0.0
-  ACME_VERSION: 2.7.4
+  CLOUDFLARE_VERSION: "2.9.11"
+  CRYPTOGRAPHY_VERSION: "41.0.5"
+  CERTBOT_VERSION: "2.7.4"
+  CERTBOT_DNS_AZURE_VERSION: "2.4.0"
+  CERTBOT_DNS_CLOUDNS_VERSION: "0.6.0"
+  CERTBOT_DNS_DESEC_VERSION: "1.2.1"
+  CERTBOT_DNS_DIRECTADMIN_VERSION: "1.0.3"
+  CERTBOT_DNS_DUCKDNS_VERSION: "1.3"
+  CERTBOT_DNS_DYNU_VERSION: "0.0.5"
+  CERTBOT_DNS_EASYDNS_VERSION: "0.1.2"
+  CERTBOT_DNS_HE_VERSION: "1.0.0"
+  CERTBOT_DNS_HETZNER_VERSION: "2.0.0"
+  CERTBOT_DNS_INFOMANIAK_VERSION: "0.2.1"
+  CERTBOT_DNS_INWX_VERSION: "2.2.0"
+  CERTBOT_DNS_JOKER_VERSION: "1.1.0"
+  CERTBOT_DNS_NAMECHEAP_VERSION: "1.0.0"
+  CERTBOT_DNS_GOOGLE_DOMAINS_VERSION: "0.1.11"
+  CERTBOT_DNS_TRANSIP_VERSION: "0.5.2"
+  CERTBOT_DNS_PORKBUN_VERSION: "0.8.0"
+  CERTBOT_DNS_WEBSUPPORT_VERSION: "2.0.1"
+  CERTBOT_GANDI_VERSION: "1.5.0"
+  CERTBOT_NETCUP_VERSION: "1.4.3"
+  CERTBOT_NJALLA_VERSION: "1.0.0"
+  CERTBOT_DNS_DREAMHOST_VERSION: "1.0"
+  CERTBOT_DNS_DOMAINOFFENSIVE_VERSION: "2.0.0"
+  CERTBOT_DNS_PLESK_VERSION: "0.3.0"
+  ACME_VERSION: "2.7.4"
```

Comment:
Can you avoid changing all those to explicit strings?

Reply:
I removed the string formats. Where can i learn, why it didn't worked without them?

Reply:
It didn't break anything, it was meant just for consistency. There is one exception where string format must be used though, and it's for versions that use numerical "X.Y" notation where "Y" is divisible by 10. In that case e.g. `1.10` would be interpreted in YAML as float value `1.1` - using quoted `"1.10"` is mandatory there.